docs/pages/guides/mesheryctl/index.md

```diff
@@ -11,10 +11,17 @@ list: exclude
 
 Guides to using Meshery's various features and components.
 
-{% capture tag %}
+{% assign sorted_guides = site.pages | sort: "name" %}
+<ul>
+  {% for item in sorted_guides %}
```

Comment:
@Vidyaa123 this doesn't seem to be working in the docs. We're missing this list, it seems.

Reply:
Never mind! I was looking at an old branch. We're all set here. 👍